src/work/SimpleDEX.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Simple Decentralized Exchange (DEX)
+ * @dev A simplified version of a DEX allowing users to swap between two tokens (TokenA and TokenB).
+ */
+contract SimpleDEX {
```

Comment:
consider adding liquidity providing methods so more liquidity can be provided for swaps

Reply:
investigating